client/src/game/ui/websocket.ts

```diff
@@ -487,9 +486,6 @@ commands.set('gameActionList', (data: GameActionListData) => {
     replay.goto(turnNum, true);
   }
 
-  cardStatusCheck();
```

Comment:
I have a feeling this might break the non-replay case

Reply:
Oh this might break shared replays, let me check

Reply:
idk it seems to work, which case do you mean?

Reply:
play a game, then in the middle of the game refresh one of the users

Reply:
seems fine

Reply:
So who calls the batchdraw functions in that case? What I'm trying to avoid is, we're "getting lucky" in a case that doesn't require the redraw.

Reply:
Ugh I understand but it works in all the cases i can think of. Is it really the job of the function that plays through all the turns to reload the UI? Sidenote, these lines are very old see [the blame](https://github.com/Zamiell/hanabi-live/blame/5298753c9677ca24027f844d252ce60c2ed74521/public/js/src/game/ui/websocket.js)